.travis.yml

```diff
@@ -1,6 +1,3 @@
-before_install:
```

Comment:
Have not realised that npm is doing the path resolving to a locally installed module when invoking npm scripts. Cheers for this!

Reply:
Also good advice is to always avoid global modules, use this in your `~/.bashrc` (or related):

```
export PATH=$PATH:node_modules/.bin:vendor/bin
```

avoids conflicts when sharing code so that everyone has same version :).

Reply:
Thanks. I'll try do do something like this, but it doesn't play nice with zsh.

I'm getting autocorrect prompts every time the command doesn't match a command from the $PATH:

```
% jake -v
zsh: correct jake to make [nyae]? n
8.0.16
% rehash
% jake -v
zsh: correct jake to make [nyae]? n
8.0.16
```

The command hash table is only built using absolute paths.

Leaving this for future reference as any of these solutions would work:

 * Globally disable correct in zsh with `unsetopt correct`
 * Use per directory zsh conf (bit hackish): https://coderwall.com/p/a3xreg/per-directory-zsh-config
 * Use direnv (my favourite solution so far) with `export PATH=$PATH:$(PWD)/node_modules/.bin` in .envrc. unsetopt is a shell built in i.e won't work in direnv without extra effort.

Reply:
Weird, I've this at the end of my `.zshrc` so that it overrides previous paths.

Reply:
Using prezto as config framework for zsh therefore it may be something specific for prezto. Have not used anything else besides prezto for the past 5 years or so. Old habits die hard :)